### ISSUE TITLE
AOR Reports: Graph Colors

### ISSUE THREAD

**Issue by daniel-samson** [salesagility/SuiteCRM#1625]
The colors of the bar graphs should be better distributed.

**Comment by gunnicom** [comment]
Maybe like this: https://github.com/salesagility/SuiteCRM/pull/1536

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [AOR Reports: Graph Colors](https://trello.com/c/WU47o0QH/93-aor-reports-graph-colors)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
